lib/commons/math/get-bounding-rect.js

```diff
@@ -0,0 +1,7 @@
+export function getBoundingRect(rectA, rectB) {
```

Comment:
JSDoc comments. Also need the `@memberof` comment to link it to the Math functions.